operator/controllers/helmrelease_controller.go

```diff
@@ -0,0 +1,62 @@
+/*
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package controllers
+
+import (
+	"context"
+
+	"github.com/go-logr/logr"
+	apierrs "k8s.io/apimachinery/pkg/api/errors"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	shipitv1beta1 "ship-it-operator/api/v1beta1"
+)
+
+// HelmReleaseReconciler reconciles a HelmRelease object
+type HelmReleaseReconciler struct {
+	client.Client
+	Log logr.Logger
+}
+
+// +kubebuilder:rbac:groups=shipit.wattpadhq.com,resources=helmreleases,verbs=get;list;watch;create;update;patch;delete
+// +kubebuilder:rbac:groups=shipit.wattpadhq.com,resources=helmreleases/status,verbs=get;update;patch
+
+func (r *HelmReleaseReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error) {
```

Comment:
Nice to see that the reconciliation is fully automated here. I am just unsure about the best way to go about running our SQS implementations in here. I.E should we import from the other module or lift and shift the Git and ECR consumers into the operator?